dedupe/record_linkage_testing/compose.yml

```diff
@@ -10,6 +10,11 @@ services:
       context: .
       dockerfile: Dockerfile.runner
     command: ["bash", "scripts/test.sh"]
+    environment:
+      - POPULATION_SIZE=100
```

Comment:
What is the motivation behind N=100?

Reply:
There's really no significance to this right now, I just needed a default N to create the variable.  10 or even 1 would have been fine too.

Eventually when the script is finished, you'll likely override that from the command line when you run the performance tests, so something like 

```sh
docker compose run --env POPULATION_SIZE=1000 runner
```